fog/sample-paykit/src/client_builder.rs

```diff
@@ -42,11 +42,11 @@ pub struct ClientBuilder {
     // Address book, for memos
     address_book: Vec<PublicAddress>,
 
-    // Optional sigstructs for attested services
-    consensus_sigstruct: Option<Signature>,
-    fog_ingest_sigstruct: Option<Signature>,
-    fog_ledger_sigstruct: Option<Signature>,
-    fog_view_sigstruct: Option<Signature>,
+    // Optional additional sigstructs for attested services
+    consensus_sigstruct: Vec<Signature>,
```

Comment:
Should these names be pluralized to match with the change?

Reply:
Yea okay